tensorflow/core/kernels/parse_tensor_op.cc

```diff
@@ -65,4 +66,33 @@ class ParseTensorOp : public OpKernel {
 
 REGISTER_KERNEL_BUILDER(Name("ParseTensor").Device(DEVICE_CPU), ParseTensorOp);
 
+
+template <typename T>
+class SerializeTensorOp : public OpKernel {
+ public:
+  explicit SerializeTensorOp(OpKernelConstruction* context)
```

Comment:
You might be able to do:

using OpKernel::OpKernel;

And avoid the need for the explicit constructor.